src/sdk/object_sdk.js

```diff
@@ -207,19 +207,17 @@ class ObjectSDK {
     async load_requesting_account(req) {
         try {
             const token = this.get_auth_token();
-            if (!token) return;
+            const is_anon = !(token && token.access_key);
```

Comment:
also here we want to only check !token and not assume empty access key is also anon, right?